### ISSUE TITLE
[Question] Not change cursor color

### ISSUE THREAD

**Issue by heartb1t** [dylanaraps/pywal#97]
Is there a way I can configure pywal to not change the cursor color? That's a little thing that's been bothering me because fonts of the same color as the cursor disappear. With the default cursor it changes its color to match the color of the current character, but the character itself appears black. Can I make that behaviour to be wal's behaviour?

**Comment by dylanaraps** [comment]
I'll look into this, thanks for reporting this.

**Comment by heartb1t** [comment]
While in the subject, is there a way to make rofi transparent with configurations? Or just through a script?

**Comment by dylanaraps** [comment]
I’m going to implement a way for users to have total control over all of
the color settings. Once this is done you’ll be able to configure rofi
however you like.

**Comment by heartb1t** [comment]
Ok, nice!

**Comment by ecly** [comment]
@heartb1t 
Till then you can consider just wrapping your run of rofi in a script such that you can add -opacity flag. Feel free to use [mine](https://github.com/ecly/dotfiles/blob/master/Scripts/rofi.sh) which is basically just default wal coloring with a border.

**Comment by heartb1t** [comment]
I've also noticed that whenever ranger updates it restores the default colors. Is that something you're aware of? Should I create another issue for you to tag it enhancement?

**Comment by heartb1t** [comment]
One more update: I've also noticed that the issue with the cursor doesn't occur in URxvt, is doesn't change the cursor color to match the character, but the character itself appears black.

**Comment by dylanaraps** [comment]
I can't reproduce the ranger issue, can you send me a screenshot?

**Comment by heartb1t** [comment]
I recorded a little gif to show you what's happening. Sorry for the delay, I have been a little bit busy.

**Comment by heartb1t** [comment]
https://gfycat.com/gifs/detail/OrderlyUnsightlyFlicker

(I may have misclicked the "Close and comment" button, sorry)

**Comment by dylanaraps** [comment]
There's no way for me to fix the cursor foreground color so I've worked around this by setting the cursor background color to `color 1`. 

I can't understand what's happening in the ranger gif (sorry!), can you maybe write some steps to reproduce the issue?

**Comment by heartb1t** [comment]
Ok, so, the steps to reproduce the ranger issue are quite simple:
* Open up ranger through the command
* Do anything that updates all of the terminal window, for example, in the gif, I opened up vim (I've tested with commands such as `:shell pandoc ...` and `:shell git ...` that, while executing, update the whole window, and for things as the the S mapping in ranger, that takes me to my shell `map S shell zsh -c "cd %d; zsh"`)
* Go back to ranger

Or, if you open up ranger with a custom mapping (for example, I have $mod+r on i3wm), ranger will not get the custom colorscheme from the start.

Expected is you would have your custom colorscheme, but what happens is that ranger sets the default colorscheme back. That might have not been clear in the gif because the custom colorscheme and the default were very alike. In the end, when I open up two different terminals, the left one is terminator and the right one is URxvt. The right one is with the custom colorscheme, but terminator reverted back to the default colorscheme - and it affects the whole terminal window - so if I leave ranger, the terminal doesn't have the custom colorscheme set anymore.

I should also mention, If I haven't yet, that this doesn't happen on URxvt, neither the ranger issue, nor the mouse thing (maybe because it uses Xresources). I can't say about other terminals, but I can say that it happens in terminator, which is the one I recorded on (it might even be a terminator issue, but I haven't got the time to test on different emulators).

I'm sorry if it was not clear what was happening. And thank you for your patience and willingness to resolve these little issues.
